### ISSUE TITLE
Update -dump behaviour (add support for streaming when go test -v)

### ISSUE THREAD

**Issue by mfridman** [mfridman/tparse#42]
I've been using `tparse` with both GitHub Actions and Drone CI and both suffer from interlaced output if too many lines are printed at once.

So, the proposal is to change the behaviour such that `tparse` streams the output (if `-dump` flag is supplied). That way whatever `go test` is printing out `tparse` regurgitates as is.

Caveat, this will always print the summary tables _at the end_ regardless of the `-top` flag. In other words, `-dump` will negate any effect `-top` has.

---

If we can no longer replay `go test` from the in-memory buffer, is there even a point to having a `-top` flag?

Reminder, `-top` will reverse the location of the summary tables, i.e., instead of printing them towards the bottom things get reversed, like so:

![image](https://user-images.githubusercontent.com/6278244/86072065-bf6c9080-ba4e-11ea-93d2-0bdae03294f2.png)

**Comment by mfridman** [comment]
I just realized GitHub actions _does not_ collapse the output and instead displays the top lines. I'm pretty sure another CI system did this, which is why I added `-top` flag, ugh.

<img width="692" alt="Screen Shot 2020-09-13 at 10 59 24 PM" src="https://user-images.githubusercontent.com/6278244/93039430-72dc2e80-f615-11ea-91c2-b6df611600c4.png">


Drone, as an example, collapses the "top" of the output and displayed only the last 200?250? lines

<img width="583" alt="Screen Shot 2020-09-13 at 10 58 57 PM" src="https://user-images.githubusercontent.com/6278244/93039416-6952c680-f615-11ea-916a-ee50d73378d3.png">

So, to answer:

    is there even a point to having a -top flag

Yes, there is. Depending on the CI and how the output is collapsed some users might want to bubble it to the top, and other might want to push it to the bottom.
